app/controllers/application_controller.rb

```diff
@@ -5,7 +5,7 @@ class ApplicationController < ActionController::Base
   include Blacklight::Controller
   layout :determine_layout if respond_to? :layout
 
-  rescue_from CanCan::AccessDenied, with: :render401
+  rescue_from CanCan::AccessDenied, with: :unauthorized
```

Comment:
I just took a quick glance at this so I might not be correct about this, but I think the error is coming from this ☝️ .  FilesController is inheriting from ApplicationController and there is no unauthorized method there.  I think this should route to the ErrorsController in some way.

Reply:
yup right, I missed that, thanks @ajkiessl @whereismyjetpack! I just added a fix.